README.md

```diff
@@ -1,5 +1,8 @@
 # LBH Cautionary Alerts API
 
+Cautionary alerts API returns information about Cautionery Alerts held
```

Comment:
```suggestion
Cautionary alerts API returns information about Cautionary Alerts held
```

Reply:
This isn't very clear. `Cautionary Alerts held` but where?
Could we use the swagger description for this API?